chore(curriculum): object and function indentation

Supravisor opened the pull request:
Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or Gitpod.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #58950

<!-- Feel free to add any additional description of changes below this line -->

Supravisor committed:
chore(curriculum): object and function indentation

ilenia-magoni commented on curriculum/challenges/english/25-front-end-development/workshop-storytelling-app/671fae2d8e402a405a930e44.md:
```diff
@@ -70,9 +70,9 @@ const adventureStoryBtn = document.getElementById("adventure-btn");
+  scary: {
+    story: `In the dark woods, a group of friends stumbled upon an old, abandoned cabin. They enter the cabin and awaken something malevolent that had been dormant for centuries.`,
+    borderColor: "#ee4b2b",
 },
```
this line is missing indentation still

Supravisor replied:
updated PR for closing brace

ilenia-magoni commented:
the function indentation is object of issue https://github.com/freeCodeCamp/freeCodeCamp/issues/58951 so you should target that too, or remove it from this PR

Supravisor committed:
fixup indentation of closing brace for first object and remove line breaks from function

Supravisor commented:
The indentation for the function was mentioned in the issue for the current PR.  Updated the "closes issue" to include the other issue number. 

> You beat me to it. 😆 Just noticed this
> 
> We should also update the formatting for the `displayStory` function because it current looks like this
> 
> function displayStory(genre) {
>     if (storyObj.hasOwnProperty(genre)) {
> 
>     result.textContent = storyObj[genre].story;
>     storyContainer.style.borderColor = storyObj[genre].borderColor;
>  
> }
> }
https://github.com/freeCodeCamp/freeCodeCamp/issues/58950#issuecomment-2677097159

Removed the line breaks from the function.

plamoni commented on curriculum/challenges/english/25-front-end-development/workshop-storytelling-app/671fafbae959c048d82bac8d.md:
```diff
@@ -141,12 +141,10 @@ const storyObj = {
- 
-}
+  if (storyObj.hasOwnProperty(genre)) {
+  result.textContent = storyObj[genre].story;
+  storyContainer.style.borderColor = storyObj[genre].borderColor;
```
The body of the if-statement should be indented from the if-statement itself (and its closing brace). I.e. you should indent these two lines by two more spaces.

plamoni commented on curriculum/challenges/english/25-front-end-development/workshop-storytelling-app/6724da4a41a727b89d2541c3.md:
```diff
@@ -78,12 +78,10 @@ const storyObj = {
- 
-}
+  if (storyObj.hasOwnProperty(genre)) {
+  result.textContent = storyObj[genre].story;
+  storyContainer.style.borderColor = storyObj[genre].borderColor;
```
Likewise, please indent these two lines by two more spaces.

plamoni commented on curriculum/challenges/english/25-front-end-development/workshop-storytelling-app/671fafbae959c048d82bac8d.md:
```diff
@@ -126,9 +126,9 @@ const adventureStoryBtn = document.getElementById("adventure-btn");
+  scary: {
+    story: `In the dark woods, a group of friends stumbled upon an old, abandoned cabin. They enter the cabin and awaken something malevolent that had been dormant for centuries.`,
+    borderColor: "#ee4b2b",
 },
```
This line should be indented by two more spaces.

ilenia-magoni commented:
> Closes #58950 & #58951

this is not going to wrok for the second issue, the "Closes" keyword goes in front of each issue

Supravisor committed:
indent if statements

Supravisor committed:
restore indentation to function and if statement

Supravisor commented:
Restored changed made to the function

plamoni commented on curriculum/challenges/english/25-front-end-development/workshop-storytelling-app/671fafbae959c048d82bac8d.md:
```diff
@@ -367,10 +367,12 @@ const storyObj = {
 function displayStory(genre) {
-  if (storyObj.hasOwnProperty(genre)) {
+    if (storyObj.hasOwnProperty(genre)) {
+
     result.textContent = storyObj[genre].story;
     storyContainer.style.borderColor = storyObj[genre].borderColor;
-  }
+ 
+}
```
This instance doesn't ever appear to have been wrong, so your PR now updates it to have incorrect indentation.

I imagine this was done accidentally when you tried to align with the current HEAD of the main branch.

Please revert the changes to these lines back to their correct indentation.

Supravisor replied:
>https://github.com/freeCodeCamp/freeCodeCamp/issues/58951 was solved by an other issue, so please remove those changes

Was asked to remove the changes. They were merged in another issue.

plamoni replied:
There were 2 places where the indentation was broken (per #58951). But the `displayStory` function actually appears in 3 places in the code. [You fixed the two broken versions](https://github.com/freeCodeCamp/freeCodeCamp/pull/58953/commits/fa4fc734f115ccf0565079c6be6ace0b967de568) and when you were asked to remove the changes related to #58951, [you un-fixed (i.e. broke) all three occurrances](https://github.com/freeCodeCamp/freeCodeCamp/pull/58953/commits/326eaeca17d5773bb4958ff2165d29fb3833b286). Because the third version was [never incorrect](https://github.com/freeCodeCamp/freeCodeCamp/blame/7b885c0c3de28214d325b02933c66063786f470e/curriculum/challenges/english/25-front-end-development/workshop-storytelling-app/671fafbae959c048d82bac8d.md#L369-L374), your change to (attempt to re-)break all three versions resulted in the third version being broken as part of your commits.

You should create another commit that reverts the change to this particular instance of `displayStory` that you made in your most recent commit.

Once you do that, you can [select the "files changed" tab](https://github.com/freeCodeCamp/freeCodeCamp/pull/58953/files) on your PR and you should not see any changes to the `displayStory` function's indentation.

ilenia-magoni replied:
to clear a doubt, reverting a change means that it should not appear in the changed files of the PR. here you did not revert a change, here you made an additional change

Supravisor replied:
> to clear a doubt, reverting a change means that it should not appear in the changed files of the PR. here you did not revert a change, here you made an additional change

Thank you for the tip.

Supravisor committed:
undo function indent

Supravisor committed:
undo changes to function indentation

Supravisor committed:
test

Supravisor committed:
2nd test

Supravisor committed:
restore function indentation in solution

Supravisor committed:
retest

Supravisor committed:
restart
